acceptance/Gemfile

```diff
@@ -11,8 +11,9 @@ def location_for(place, fake_version = nil)
 end
 
 gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 0.2")
-gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 2.8')
+gem "beaker", '~> 2.5'
```

Comment:
@johnduarte Im not sure what the best practice is here.  Im not sure if the current pipelines emit the BAEKER_TYPE environment variable.